docs/providers/ai21.md

```diff
@@ -9,32 +11,44 @@ AI21_API_KEY=
 {% endcode %}
 
 ```typescript
-import { TokenJS, ChatCompletionMessageParam } from 'token.js'
+import { TokenJS } from 'token.js'
 
-// Import and create the token.js client
+// Create the Token.js client
 const tokenjs = new TokenJS()
 
-// Specify OpenAI compatible messages
-const messages: ChatCompletionMessageParam = [{
-  role: 'user',
-  content: `How are you?`,
-}]
-
-// Call the create function
-const result = await tokenjs.chat.completions.create({
-  // Specify the target model and provider
-  provider: 'ai21',
-  model: 'jamba-instruct',
-  messages,
-})
```

Comment:
This does not cause a type error, at least not for me. 
<img width="431" alt="Screenshot 2024-07-13 at 12 04 36 PM" src="https://github.com/user-attachments/assets/ecd85141-4a04-41b7-98f2-fa5f0906773b">

Reply:
The purpose of defining the messages array separately was to demonstrate the use of types and that you can import the types from our package which is a nice quality of life thing IMO. 

I don't think removing it benefits the docs, but I'm not super opinionated on it. I just think removing this makes them slightly worse.

Reply:
I'm not very opinionated on this either, but I'm still getting a type error, so I don't think we should consider including it in the example until that's resolved. I opened an issue for it here: https://github.com/token-js/token.js/issues/67

Want me to create a separate ticket adding the `ChatCompletionsMessageParam` type back to the documentation examples?